src/Core/ExecutionPathTracer/Register.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.IQSharp
+{
+    /// <summary>
+    /// Enum for the 2 types of registers: Qubit and Classical.
+    /// </summary>
+    public enum RegisterType
+    {
+        Qubit,
+        Classical,
```

Comment:
Since this is a public enum, some API documentation comments would be really helpful here.